docs/docs-content/vm-management/environment-setup.md

```diff
@@ -0,0 +1,161 @@
+---
+sidebar_label: "Environment Setup"
+title: "Environment Setup"
+description: "Learn about Palette VMO pack and the architecture behind it."
+hide_table_of_contents: false
+sidebar_position: 5
+tags: ["vmo", "environment-setup"]
+---
+
+This page provides reference specifications for the Palette Virtual Machine Orchestrator (VMO) hardware and network
+resources.
+
+## Hardware Resources
+
+Refer to the following table for the minimum and recommended hardware specifications for the worker nodes of the
+cluster.
+
+| Component            | Minimum                                            | Recommended                                        | Comments                                                                         |
+| -------------------- | -------------------------------------------------- | -------------------------------------------------- | -------------------------------------------------------------------------------- |
+| **Server**           | 2U Rackmount Chassis                               | 2U Rackmount Chassis                               | Needs to fit FC adapters and have sufficient NICs.                               |
+| **CPU**              | Intel or AMD x64 CPU with 8 cores                  | Intel or AMD x64 CPU with 8 cores                  |                                                                                  |
+| **RAM**              | 24 GB                                              | 256 GB or more                                     | Assumes the deployment of 20 VMs per node multiplied by the median RAM per VM.   |
+| **Network Adapters** | 2 x 10 Gbps <br /> (data + management)             | 2 x 10 Gbps (data) <br /> 2 x 10 Gbps (management) | Pod overlay operates on the management network.                                  |
+| **Storage Adapters** | 2 x 16 Gbps FC                                     | 2 x 16 Gbps FC                                     |                                                                                  |
+| **Disks**            | Local disk for the OS boot (SAN boot is supported) | Local disk for the OS boot                         | Boot from SAN requires special consideration due to the multipath configuration. |
+
+Typically, the cluster control plane nodes do not operate any VMO workloads. As a result, they can have lighter hardware
+specifications. For example, a server with 4 cores and 8 GB RAM is sufficient for a minimum-specification control plane
+node.
```

Comment:
In the table above, we state that 8 cores are the minimum. Are  four cores the minimum then?